Microsoft.Toolkit.Uwp.Input.GazeInteraction/GazeHidParsers.h

```diff
@@ -0,0 +1,128 @@
+#pragma once
+
+#include "GazeHidUsages.h"
+
+using namespace Windows::Devices::HumanInterfaceDevice;
+using namespace Windows::Devices::Input::Preview;
+
+BEGIN_NAMESPACE_GAZE_INPUT
+
+namespace GazeHidParsers {
+    public ref class GazeHidPosition sealed
+    {
+    private:
+        long long _X;
+        long long _Y;
+        long long _Z;
+
+    public:
+        property long long X
+        {
+            long long get() {
+                return _X;
+            }
+            void set(long long value) {
+                _X = value;
+            }
+        }
+
+        property long long Y
+        {
+            long long get() {
+                return _Y;
+            }
+            void set(long long value) {
+                _Y = value;
+            }
+        }
+
+        property long long Z
+        {
+            long long get() {
+                return _Z;
+            }
+            void set(long long value) {
+                _Z = value;
+            }
+        }
+    };
+
+    class GazeHidParsers {
```

Comment:
This looks like it could be a static function in the .cpp (or a class entirely declared in the .cpp); doesn't seem to be part of the .h.